proto/stream_plan.proto

```diff
@@ -530,21 +530,13 @@ message StreamActor {
   string mview_definition = 9;
 }
 
-enum FragmentType {
-  FRAGMENT_UNSPECIFIED = 0;
-  OTHERS = 1;
-  SOURCE = 2;
-  // TODO: change it to MATERIALIZED_VIEW or other name, since we have sink type now.
-  SINK = 3;
-}
-
 message StreamFragmentGraph {
   message StreamFragment {
     // 0-based on frontend, and will be rewritten to global id on meta.
     uint32 fragment_id = 1;
     // root stream node in this fragment.
     StreamNode node = 2;
-    FragmentType fragment_type = 3;
+    uint32 fragment_type_mask = 3;
```

Comment:
I'm afraid that defining the flags completely in the Rust code will make the compatibility harder in the future. However, I have no idea of a better solution.

Reply:
How about define the flag types in proto as follows:
```proto
enum FragmentTypeFlag {
  UNSPECIFIED = 0x0;
  SOURCE = 0x01;
  MVIEW = 0x02;
  SINK = 0x04;
}

message StreamFragment {
... ...
  // Bitwise-OR of Flags
  uint32 type_mask = 1;
... ...
}
```

Reply:
> How about define the flag types in proto as follows:
> 
> ```proto
> enum FragmentTypeFlag {
>   UNSPECIFIED = 0x0;
>   SOURCE = 0x01;
>   MVIEW = 0x02;
>   SINK = 0x04;
> }
> 
> message StreamFragment {
> ... ...
>   // Bitwise-OR of Flags
>   uint32 type_mask = 1;
> ... ...
> }
> ```

I'm very willing to do that. What do you think? @BugenZhao

Reply:
+1 for this. 🤤

Reply:
> How about define the flag types in proto as follows:
> 
> ```proto
> enum FragmentTypeFlag {
>   UNSPECIFIED = 0x0;
>   SOURCE = 0x01;
>   MVIEW = 0x02;
>   SINK = 0x04;
> }
> 
> message StreamFragment {
> ... ...
>   // Bitwise-OR of Flags
>   uint32 type_mask = 1;
> ... ...
> }
> ```

I have just modified code to moved to this :-)